monkeys/fqdn-s3/values-fqdn-s3.yaml

```diff
@@ -0,0 +1,7 @@
+name: "fqdn-s3"
+replicas: 3
+fqdn: "s3.amazonaws.com"
+env:
+    sleep: "1"
+    connectTimeout: "5"
```

Comment:
This might be exactly 1 DNS timeout. I'm not sure which is better to test: that we have no DNS drops, or that DNS succeeded at all, after 2 retries (that would mean a value >10 here).